spec/datadog/tracing/contrib/elasticsearch/transport_spec.rb

```diff
@@ -14,7 +14,7 @@
     # TODO: Find why using testagent changes the elasticsearch test service name despite no differences between other tests
     Datadog.configuration.agent.host = 'not-testagent'
     WebMock.enable!
-    WebMock.disable_net_connect!
+    WebMock.allow_net_connect!
```

Comment:
This seems reasonable but... is it only weird for me that we start telemetry during random specs?

Should we have a more generic "don't run telemetry during testing"?

(Also -- does this change impact the TODOs on line 13 and 14?)

Reply:
Should we remove the `WebMock.allow_net_connect!` from the `after` block?

Reply:
> Should we remove the `WebMock.allow_net_connect!` from the `after` block?

Not today.
This combination of webmock settings is how our spec_helper sets up all tests:
https://github.com/DataDog/dd-trace-rb/blob/d6fcf015a2eeaa7f9a853378d70bc757fc8a8f71/spec/spec_helper.rb#L60-L61

Reply:
> This seems reasonable but... is it only weird for me that we start telemetry during random specs?
> 
> Should we have a more generic "don't run telemetry during testing"?
> 
> (Also -- does this change impact the TODOs on line 13 and 14?)

Agreed!